ledger-core/application/benchs/benchrunner_test.go

```diff
@@ -132,28 +131,22 @@ func runBench(iterations int, name string, workerFunc func(ctx context.Context,
 	startBench := time.Now()
 	counterStopped := make(chan struct{})
 
-	// I don't use ticker, because with a lot of goroutines it cannot handle the case
 	go func() {
+		reportTicker := time.NewTicker(time.Second)
```

Comment:
I wonder why you removed my comment and reverted this change

Reply:
I've run that with ticker, and there have not been problems